action.yml

```diff
@@ -21,7 +21,7 @@ inputs:
     required: false
     default: 'master'
   github_token:
-    description: 'A github token passed from secrets'
+    description: GitHub TOKEN or PAT
```

Comment:
TOKEN is understandable. PAT??? What does it mean?

Reply:
PAT is an abbreviation of Personal Access Token - https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens